spec/models/manageiq/providers/hawkular/middleware_manager/refresh_parser_spec.rb

```diff
@@ -2,25 +2,22 @@
 require 'recursive-open-struct'
 
 describe ManageIQ::Providers::Hawkular::MiddlewareManager::RefreshParser do
-  let(:ems_hawkular) do
-    # allow(MiqServer).to receive(:my_zone).and_return("default")
-    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+  before do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
     auth = AuthToken.new(:name => "test", :auth_key => "valid-token", :userid => "jdoe", :password => "password")
-    FactoryGirl.create(:ems_hawkular,
-                       :hostname        => 'localhost',
-                       :port            => 8080,
-                       :authentications => [auth],
-                       :zone            => zone)
+    @ems_hawkular = FactoryGirl.create(:ems_hawkular,
+                                       :hostname        => 'hservices.torii.gva.redhat.com',
```

Comment:
I copied from here: https://github.com/josejulio/manageiq/blob/d4af06419d5e5538114f10258b87c589ad3724d1/spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb#L6
Though it could be useful when re-recording so I used the same. But I'm on the process of removing the changes on this file because I'm moving the logic a bit.